core-bundle/src/Command/ResizeImagesCommand.php

```diff
@@ -107,6 +108,7 @@ protected function configure(): void
             ->addOption('throttle', 't', InputOption::VALUE_OPTIONAL, '(Deprecated) Use the concurrent option instead', false)
             ->addOption('image', null, InputArgument::OPTIONAL, 'Image name to resize a single image')
             ->addOption('no-sub-process', null, InputOption::VALUE_NONE, 'Do not start a sub process per resize')
+            ->addOption('delete-missing', null, InputOption::VALUE_NONE, 'Delete deferred image references to images that no longer exist')
```

Comment:
I still think it should be the opposite and missing should delete by default.

Reply:
@contao/developers WDYT?

Reply:
What is the motivation to keep references to images that no longer exist?

Reply:
If the image is only missing temporarily.

Reply:
I think I am with @Toflar on this one. Delete by default but provide a `--preserve-missing` or `--no-deletes` flag for these special situations.

Reply:
Done in b947f1158095e585cdd1404b499a2dbe1db2c4e9